.gitmodules

```diff
@@ -34,3 +34,15 @@
 [submodule "modules/DroneCAN/libcanard"]
 	path = modules/DroneCAN/libcanard
 	url = https://github.com/DroneCAN/libcanard
+[submodule "modules/uavcan_v1/libcanard"]
+	path = modules/uavcan_v1/libcanard
+	url = https://github.com/UAVCAN/libcanard.git
+[submodule "modules/uavcan_v1/o1heap"]
+	path = modules/uavcan_v1/o1heap
+	url = https://github.com/pavel-kirienko/o1heap.git
```

Comment:
why do we need this ? doesn't UAVCAN have this on its own repo ?

Reply:
No. Here we have the original repository of o1heap. There is nothing same inside uavcan organization. This repo link I got from the example from the libcanard readme.